benchmark/cajita/Cajita_FastFourierTransformPerformance.cpp

```diff
@@ -151,10 +151,10 @@ int main( int argc, char* argv[] )
 
     // Declare the grid size per dimension
     // currently, testing 3dims+symmetric
-    std::vector<double> grid_sizes_per_dim = { 10, 20 };
+    std::vector<double> grid_sizes_per_dim = { 16, 32 };
```

Comment:
Unrelated change?

Reply:
We were just trying to make things consistent. Probably need a PR to update the FFT benchmark anyway though